STYLEGUIDE.md

```diff
@@ -1,39 +1,311 @@
+
+#### Don't
+
+> Now that Clerk is installed and mounted in your application, you can decide which pages are public and which should require authentication to access.
+
+### 2.4 Avoid gerunds (-ing words)
+
+English gerunds ("-ing" words like "running") turn verbs into nouns ("run" becomes "running"). This makes the sentence sound passive ("They run" becomes "they are running") and makes it harder to translate. Use an active voice as much as possible and avoid these words.
+
+#### Do
+
+> How to use a routing library with Clerk
+
+#### Don't
+
+> Using a routing library with Clerk
+
+### 2.5 Use task and outcome-based headings and titles that use verbs.
+
+Tell the learner what they will learn from diving into the content. 
+
+#### Do
+
+> How to read session and user data
+
+> How user sessions and user data work
+
+Clearly explains what information the learner will find, helping them decide which to choose.
+
+#### Don't
+
+> User session and user data
+
+Obscure. Doesn't convey whether this is the conceptual or task-based content the learner is looking for. They may pass over it or look briefly, then bounce out. This increases friction.
+
+### 2.6 Lead with location; end with action.
+
+When learners are performing an order of operations, it helps for them to start with _where_ they need to be to perform the action.
+
+#### Do
+
+> In your Next.js project's root folder, open your `.env.local file`.
+
+The learner navigates to the project folder first, opens the file second.
+
+#### Don't
+
+> Open your `.env.local file` in your Next.js project's folder.
+
+The learner must remember the file they need to open before finding out which folder to navigate to. This uses more mental energy.
+
+### 2.7 Avoid colloquialisms
+
+English speakers my find themselves using phrases like "
+
+#### Do
+
+> Something that is difficult for someone new to programming may not be for a senior engineer.
+
+#### Don't
+
+> Something that is difficult for someone new to programming may be child's play for a senior engineer.
+
+Some learners may not have heard this expression before. It may be difficult for machine translation as well.
+
+## 3. Write to include people
+
+### 3.1 Do not assume proficiency
+
+Avoid using language that assumes someone's level of proficiency. Something that is difficult for someone new to programming may not be difficult for a senior engineer. This language can inadvertently alienate or insult a learner. Avoid words like "just," "easy," "simple," "senior," "hard."
+
+#### Do
+
+> You can authenticate your app with Clerk in three steps. Install Clerk with `npm install @clerk/nextjs`, add your environment keys, and then wrap your app in `<ClerkProvider />`, and add [control components](https://clerk.com/docs/components/overview). Visit our [Quickstarts](https://clerk.com/docs/quickstarts/overview) for a step-by-step guide written for your framework.
```

Comment:
```suggestion
> You can authenticate your app with Clerk in three steps. Install Clerk with `npm install @clerk/nextjs`, add your environment keys, and then wrap your app in `<ClerkProvider />`. Visit our [Quickstarts](https://clerk.com/docs/quickstarts/overview) for a step-by-step guide written for your framework.
```

Reply:
We're using "control components" somewhere... Are we not using that terminology anymore?